content/about.md

```diff
@@ -1,28 +1,33 @@
 --- 
 doc-note-type: draft
-title: About Supplemental Guidance
+title: About Supplemental Guidance to WCAG 2
```

Comment:
I thought we recently removed this?

Reply:
OK ! Sorry, brain fart.

In the header, I wonder if we want to add "2" after WCAG (since it's not WCAG 3)?